Wbooks/Library/LibraryController.swift

```diff
@@ -12,30 +12,22 @@ final class LibraryController: UIViewController, UITableViewDelegate, UITableVie
     
     // MARK: Properties
     private lazy var libraryView: LibraryView = LibraryView()
-    private let booksArray : [Book] = [
-        Book(cover: UIImage(named: "BookCover1")!,
-             title: "A Little Bird Told Me",
-             author: "Timothy Cross"),
-        Book(cover: UIImage(named: "BookCover2")!,
-             title: "When the Doves Disappeared",
-             author: "Sofi Oksanen"),
-        Book(cover: UIImage(named: "BookCover3")!,
-             title: "The Best Book in the World",
-             author: "Peter Sjernstrom"),
-        Book(cover: UIImage(named: "BookCover4")!,
-             title: "Be Creative",
-             author: "Tony Alcazar"),
-        Book(cover: UIImage(named: "BookCover5")!,
-             title: "Redesign the Web",
-             author: "Liliana Castilla")]
+    private var viewModel = LibraryViewModel()
+    
+    // MARK: Initializers
+    init(viewModel: LibraryViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: .none, bundle: .none)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
     
     // MARK: Lifecycle methods
     override func viewDidLoad() {
         super.viewDidLoad()
-        let nib = UINib(nibName: "LibraryCell", bundle: nil)
-        libraryView.booksTable.register(nib, forCellReuseIdentifier: LibraryCell.identifier)
-        libraryView.booksTable.delegate = self
-        libraryView.booksTable.dataSource = self
+        setupDelegates()
```

Comment:
Nice refactor! 😉  Extracting code into methods is always a good idea for keeping things simple. 
However this method doing more than just setting up the delegates 🤔  so perhaps we could call it `setupView`? Or something like that?